ipatests/test_integration/test_backup_and_restore.py

```diff
@@ -861,3 +874,43 @@ def test_replica_install_after_restore(self):
         # install second replica after restore
         tasks.install_replica(master, replica2)
         check_replication(master, replica2, "testuser2")
+
+
+class TestBackupAndRestoreTrust(IntegrationTest):
+    """Test for Backup and Restore for Trust scenario"""
+    topology = 'star'
+
+    def test_restore_trust_pkg_before_restore(self):
+        """Test restore with adtrust when trust-ad pkg is missing.
+
+        Test for bug https://pagure.io/freeipa/issue/7630.
+        """
+        pkg_installer = tasks.pkg_installer(self.master)
+        self.master.run_command([pkg_installer, '-y',
+                                 'install', '*ipa-server-trust-ad'])
```

Comment:
The mthod tasks.install_packages(self.master, ['*ipa-server-trust-ad']) can be used instead.